src/main/java/edu/pw/eiti/pik/participation/ParticipationController.java

```diff
@@ -33,36 +25,13 @@ public ParticipationController(ParticipationService participationService) {
     @ApiOperation(value = "Change participation status",
             notes = "Allow current user to change participation status in current project")
     @PostMapping(path = "/participation/{status}/{projectId}")
-    void changeStatus(@PathVariable String status, @PathVariable Long projectId,
```

Comment:
That only won't work.
https://machiel.me/post/java-enums-as-request-parameters-in-spring-4/